From 9702507a04a416f41ecbbe14ffed28d48ae1d63a Mon Sep 17 00:00:00 2001
From: fupan li <fupan.li@windriver.com>
Date: Fri, 22 Aug 2014 14:56:37 +0800
Subject: [PATCH] fix a bug in drivers/gpio/Kconfig

commit 0571734bf05ea2f4683942b06b7070214986e749
introduce a bug in drivers/gpio/Kconfig where two "space" was missing on
line 170 and 171.

This give us this log when doing make -C build linux-windriver.menuconfig
ERROR: Function failed: do_configure (see
/buildarea4/idp-nightly-log/tnie_idp_1408211319/gigbyte_glibc-idp_allfeatur
e_srm_grsec_2/bitbake_build/tmp/work/intel_atom_baytrail-wrs-linux/linux-wi
ndriver-3.4-r0/temp/do_configure/log.do_configure.11040 for further
information)
ERROR: Logfile of failure stored in:
/buildarea4/idp-nightly-log/tnie_idp_1408211319/gigbyte_glibc-idp_allfeatur
e_srm_grsec_2/bitbake_build/tmp/work/intel_atom_baytrail-wrs-linux/linux-wi
ndriver-3.4-r0/temp/do_configure/log.do_configure.11040
Log data follows:
| DEBUG: Executing python function sysroot_cleansstate
| DEBUG: Python function sysroot_cleansstate finished
| DEBUG: Executing shell function do_configure
| NOTE: make oldconfig
|   GEN
/buildarea4/idp-nightly-log/tnie_idp_1408211319/gigbyte_glibc-idp_allfeatur
e_srm_grsec_2/bitbake_build/tmp/work/intel_atom_baytrail-wrs-linux/linux-wi
ndriver-3.4-r0/linux-intel-atom-baytrail-standard-build/Makefile
| scripts/kconfig/conf --oldconfig Kconfig
| drivers/gpio/Kconfig:171: syntax error
| drivers/gpio/Kconfig:170: unknown option "To"
| drivers/gpio/Kconfig:171: unknown option "be"
| make[3]: *** [oldconfig] Error 1
| make[2]: *** [oldconfig] Error 2
| make[1]: *** [sub-make] Error 2
| make: *** [all] Error 2
| ERROR: oe_runmake failed

The fix was to add 2 spaces char in line 170 and 171 to align everything.

Signed-off-by: Arnaud Blanchard <Arnaud.Blanchard@windriver.com>
Signed-off-by: fupan li <fupan.li@windriver.com>
---
 drivers/gpio/Kconfig |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/Kconfig b/drivers/gpio/Kconfig
index 3c830af..8610ce7 100644
--- a/drivers/gpio/Kconfig
+++ b/drivers/gpio/Kconfig
@@ -167,8 +167,8 @@ config GPIO_SCH311X
 	  Driver to enable the GPIOs found on SMSC SMSC SCH3112, SCH3114 and
 	  SCH3116 "Super I/O" chipsets.
 
-	To compile this driver as a module, choose M here: the module will
-	be called gpio-sch311x.
+	  To compile this driver as a module, choose M here: the module will
+	  be called gpio-sch311x.
 
 config GPIO_XILINX
 	bool "Xilinx GPIO support"
-- 
1.7.5.4

